src/screens/Sidebar/OrgSidebar.res

```diff
@@ -68,12 +68,11 @@ module OrgTile = {
     let displayText = {
       let firstLetter = orgName->String.charAt(0)->String.toUpperCase
       if orgName == orgID {
-        let count =
-          orgList
-          ->Array.slice(~start=0, ~end=index + 1)
-          ->Array.filter(org => org.name == org.id)
-          ->Array.length
-        `O${count->Int.toString}`
+        let lastTwoChars =
+          orgID
+          ->String.slice(~start=orgID->String.length - 2, ~end=orgID->String.length)
+          ->String.toUpperCase
+        lastTwoChars
```

Comment:
can directly return instead of assigning a variable name as it is straightforward single line logic